### PR TITLE
Add OS summaries

### PR DESCRIPTION
**Pull request by dpb587-pivotal**
Useful? I'd imagine this would be a good place for a more explicitly defined support policy, too. And maybe a couple other sections of information, or defining package sets at some point?

### CODE REVIEW

**Comment by freddesbiens** [comment]
This is very good, @dpb587-pivotal! The one comment I have is about Windows. The actual name for the products we support is not just "Windows" but rather "Windows Server" (as opposed to the Windows Client OS). You should replace "Windows 2016" with "Windows Server 2016" and "Windows 2012R2" with "Windows Server 2012 R2".

**Comment by mfine30** [comment]
This is awesome Danny. Definitely supportive of merging and publishing

File: content/ubuntu-os.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Stemcell OS: Ubuntu
+
+The base Ubuntu operating system is built from the [cloudfoundry/bosh-linux-stemcell-builder](https://github.com/cloudfoundry/bosh-linux-stemcell-builder) repository.
+
+The most recent versions of these distributions are built every other week to ensure the latest upstream patches are included. Critical and High CVEs result in new builds, regardless of the biweekly interval. These builds are published on [bosh.io](https://bosh.io/) as new, minor versions.
```
Comment (voelzmo) [inline content/ubuntu-os.md]: Maybe that's just me as a non-native speaker: "every other week" is less clear than "every two weeks". Can we change that? Or maybe that's just me?

File: content/ubuntu-os.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+!!! tip
+    [Ubuntu Xenial](#ubuntu-xenial) is the recommended Linux distribution for deploying with BOSH.
```
[the remark this answers is not in the document]
Comment (dpb587-pivotal) [reply content/ubuntu-os.md]: I think they still technically consider it experimental. From BOSH's perspective though, I'd like to start recommending it though. I think once bosh-deployment starts referring to it by default we'd mention it more outside of this new, less-traveled page.

File: content/windows-os.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+### Windows Server 2012 R2
+
+You can find the official stemcells from [bosh.io/stemcells](https://bosh.io/stemcells#windows2012R2).
+
+
+## Licensing
+
+Windows stemcells will have additional costs associated with Microsoft licensing. Windows stemcell do not include actual Windows OS. When using light stemcells on public clouds, additional costs will be associated with your virtual machine. For more information on building Windows stemcells and running in on-premise environments, please see this [Getting Started](https://github.com/cloudfoundry-incubator/bosh-windows-stemcell-builder/wiki/BOSH-Windows-Getting-Started-Guide) guide.
```
[the remark this answers is not in the document]
Comment (dpb587-pivotal) [reply content/windows-os.md]: Fixed; thanks

File: content/ubuntu-os.md (hunk 1 above)
Comment (dpb587-pivotal) [reply content/ubuntu-os.md]: Improved; thanks

**Pull request merged**